content/docs/user-guide/programming/components/reflection/serialization-context/versioning.md

```diff
@@ -1,7 +1,6 @@
 ---
-description: ' Use the Open 3D Engine component serialization versioning system to validate
-  and update the serialized data of your components. '
 title: Versioning your Component Serialization
+description: Use the Open 3D Engine component serialization versioning system to validate and update the serialized data of your components.
```

Comment:
Doh! I missed "your" in the title. Per title case capitalization, nouns/pronouns also get capitalized. Basically, capitalize everything except articles ("a"/"an"/"the") and prepositions. So that line should be:

>title: Versioning Your Component Serialization